CMakePresets.json

```diff
@@ -0,0 +1,307 @@
+{
+    "version": 3,
+    "configurePresets": [
+        {
+            "hidden": true,
+            "name": "linux",
+            "condition": {
+                "type": "equals",
+                "lhs": "${hostSystemName}",
+                "rhs": "Linux"
+            }
+        },
+        {
+            "hidden": true,
+            "name": "ci-base",
+            "displayName": "CI configuration",
+            "description": "Sets Ninja generator, build and install directory",
+            "generator": "Ninja",
+            "binaryDir": "${sourceDir}/out/build/${presetName}",
+            "installDir": "${sourceDir}/out/install/${presetName}",
+            "cacheVariables": {
+                "CMAKE_BUILD_TYPE": "Release",
+                "BLA_VENDOR": "Generic"
+            },
+            "warnings": {
+                "dev": true,
+                "deprecated": true
+            }
+        },
+        {
+            "name": "default",
+            "displayName": "Default build configuration",
+            "description": "The right build configuration for most users",
+            "inherits": "ci-base",
+            "cacheVariables": {
+                "BUILD_TESTING": false,
+                "CMAKE_COMPILE_WARNING_AS_ERROR": false
+            },
+            "warnings": {
+                "dev": false,
+                "deprecated": false
+            }
```

Comment:
Warnings (CMake and compiler) in CI/dev-default vs default: Builds for regular users (who don't intend to develop/modify Simbody itself) should be quiet (no warnings); only hard errors should be shown. Compiling unfamiliar software is difficult and a wall of irrelevant text doesn't make anything easier. 

This doesn't silence compiler warnings yet; I'm considering adding a "DEVELOP_SIMBODY" CMake variable to use to turn on some dev defaults in a more cross platform way (instead of needing different presets to silence warnings correctly in different compilers).